### PR TITLE
Added me as author, lol

### PR DESCRIPTION
**Pull request by jamal-0x1**
Added me as an author, because why not?

### CODE REVIEW

**Comment by IsraPerez98** [comment]
LGTM
Can I be added as an author as well?

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala [hunk 1]
```diff
@@ -226,6 +226,12 @@ object HomeTweetTypePredicates {
       candidate =>
         candidate
           .getOrElse(AuthorIdFeature, None).contains(candidate.getOrElse(DDGStatsElonFeature, 0L))),
+    (
+      "author_is_HeyJamalUddin",
```
Comment (CoolTechYT) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:230]: Need to add another author "author_is_CoolTechYT", will approve after

**Comment by angrymouse** [comment]
WAGMI! I want to be very personal author as well!

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala [hunk 2]
```diff
@@ -226,6 +226,12 @@ object HomeTweetTypePredicates {
...
+      candidate =>
+        candidate
+          .getOrElse(AuthorIdFeature, None)
+          .exists(candidate.getOrElse(DDGStatsDemocratsFeature, Set.empty[Long]).contains)),
```
Comment (bwedding) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:234]: All you did was classify yourself as a democrat, which you probably already are, bozo.

Comment (jamal-hex) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:234]: Oh, fuck! Will change that, Mr. Old Timer!

**Pull request closed without merging**